Common/PkmnModels/Status.cs

```diff
@@ -0,0 +1,33 @@
+using System.Runtime.Serialization;
+
+namespace Common.PkmnModels
+{
+    [DataContract]
+    public struct NonvolatileStatus
+    {
+        [DataMember(Name = "brn")] public bool Brn { get; set; }
+        [DataMember(Name = "frz")] public bool Frz { get; set; }
+        [DataMember(Name = "par")] public bool Par { get; set; }
+        [DataMember(Name = "psn")] public bool Psn { get; set; }
+        [DataMember(Name = "slp")] public int Slp { get; set; }
+        [DataMember(Name = "tox")] public int Tox { get; set; }
+    }
+
+    [DataContract]
+    public struct VolatileStatus
+    {
+        [DataMember(Name = "cnf")] public int Cnf { get; set; }
+        [DataMember(Name = "cur")] public bool Cur { get; set; }
+        [DataMember(Name = "foc")] public bool Foc { get; set; }
+        [DataMember(Name = "inf")] public bool Inf { get; set; }
+        [DataMember(Name = "tau")] public bool Tau { get; set; }
+        [DataMember(Name = "tor")] public bool Tor { get; set; }
```

Comment:
I've never seen volatiles except *maybe* confusion be referred to with these 3-letter abbreviations. I can't even guess at what "tau" and "tor" are. Volatile statuses are usually given their full name. This isn't an exhaustive list, either, from what I can tell.

Reply:
I don't know either, I just took whatever pbrengine spat out and put it here 😄
Would making this list exhaustive and giving the things descriptive code names be okay with you? Changing the representations is a bit annoying, since I'd need to hunt down and change all occurance in pbrengine, the old core and the overlay

Reply:
Are they even used in the core/overlay currently? I can't think of a place where they would be, except *maybe* in run API, but that's not pbrengine anyways

If we are going to have a use case for these then yes, it's probably worth having any volatile status here. Side effects and field effects, too, why not. Showdown almost always identifies them by their respective move/ability/item name (so "attract" instead of "infatuated")

Reply:
yes, "tau" is used in the calculation of bezier curves, next to "sin" and "pi".

jokes aside, it seems to only use nonvolatile statuses:
```javascript
for (var key in pokeset.status.nonvolatile) {
    let val = pokeset.status.nonvolatile[key];
    if (val) {
        st_url = "/static/status/bg/" + key + ".jpg";
        $sprite.css({"background-image": 'url("' + st_url + '")'});
        nv_count += 1;
    }
}
```

also I found this in the current match data update code:
```python
"status": {
    "nonvolatile": deepcopy(mon["status"]["nonvolatile"]),
    # "volatile": deepcopy(mon["status"]["volatile"]),  # doesn't seem to get updated by pbrengine
},
```

so I think we can safely remove volatile stuff until we actually need it

Reply:
If PbrEngine is using confusing three-letter abbreviations, then maybe fix _that_ instead of propagating a bad standard?

Reply:
It isn't, those are currently set by pokecat. to quote ferraro from discord:
> I added those fields to pokecat during pokeset population, see init line 408
> I was also not really sure how to handle those fields though, since they do seem to relate to a particular pokemon instance

The above code was a direct depiction of what went "over the wire", as a starting point and to be able to discuss how things currently are.

Also, sorry if that's not how you wanted to sound, but please don't just comment here to drop superficial passive-aggressive oneliners.

Reply:
I get flooded by emails whenever threads like this one get going. Don't blame me for actually checking them out.
That being said, if you know the source of those confusing labels, it's probably best to start there (wherever that source may be) and fix this from the source.

If you think my suggestions are "passive-aggressive one-liners", I'd strongly recommend taking a short break; you're reading stuff that isn't there.